app/main.py

```diff
@@ -2,0 +1,11 @@
+def copy_file(command: str) -> None:
+    split_command = command.split()
+    if split_command[1] == split_command[2]:
+        return
```

Comment:
The function silently does nothing when the source and destination files are the same. It would be better to inform the user about this, for example by raising an exception or returning an error message.